client/elements/text/sc-bilara-segmented-text.js

```diff
@@ -140,11 +144,26 @@ class SCBilaraSegmentedText extends SCLitTextPage {
       <sc-bottom-sheet></sc-bottom-sheet>
     `;
   }
-
+  
   firstUpdated() {
+    // location-changed event is added by iron-location component.
+    // iron-location disables default behavior of hash-links and re-emits a different event.
+    // hashchange can still fire, but only due to browser back & forward buttons.
+    // TODO: In case iron-location is removed, this will break, but it's not possible to bypass it now

```

Comment:
If there is any issue which you're not able to fix right now, create an issue and add here at least an gihub id.